assets/models/system/databricks-dolly-v2-3b/spec.yaml

```diff
@@ -29,4 +29,4 @@ tags:
       
 
   task: text-generation
-version: 1
+version: 2
```

Comment:
Where can I find other files viz.; model.yaml, description etc?

Reply:
As we have added this model for first time, I have run 2 times this model one without Fine tune of Token classification and one with Finetune of Token classification.  

First PR of it is in  https://github.com/Azure/azureml-assets/pull/1161/commits/8f27344de0948d108b37f96c291255e6ba9bfbd6


Second PR is this that is why I changed the Version to 2